src/index.html

```diff
@@ -11,7 +11,133 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://localhost:8080" method="post">
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <label class="block">
+          Surname:
+          <input type="text" name="surname">
```

Comment:
Text fields should have autocomplete="off".
Every field should have name attribute.
Please fix for all cases.